ast/src/builder.rs

```diff
@@ -22,6 +22,23 @@ impl Repo {
         self.build_graph_inner().await
     }
     pub async fn build_graph_inner<G: Graph>(&self) -> Result<G> {
+        #[cfg(feature = "neo4j")]
+        {
```

Comment:
to keep this function clean, can you move this code into a separate util function